src/resources/transactions-resource.js

```diff
@@ -78,5 +78,9 @@ export default function TransactionsResource({apiHandler}) {
         createTimelineComment({id, data}) {
             return apiHandler.create(`transactions/${id}/timeline`, '', data);
         },
+
+        patch({id, data}) {
```

Comment:
I'd put it after `create` method https://github.com/Rebilly/rebilly-js-sdk/pull/216/files#diff-d8ac3302ccaf17aa9ab1d96f98db82d9R43